api/cloud-resources/v1beta1/awsnfsvolume_types.go

```diff
@@ -113,6 +113,32 @@ func (in *AwsNfsVolume) SpecificToProviders() []string {
 	return []string{"aws"}
 }
 
+func (in *AwsNfsVolume) GetIpRangeRef() IpRangeRef {
+	return in.Spec.IpRange
+}
+
+func (in *AwsNfsVolume) State() string {
+	return in.Status.State
+}
+
+func (in *AwsNfsVolume) SetState(v string) {
+	in.Status.State = v
+}
+
+func (in *AwsNfsVolume) CloneForPatchStatus() client.Object {
+	return &AwsNfsVolume{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "AwsNfsVolume",
+			APIVersion: GroupVersion.String(),
+		},
```

Comment:
If TypeMeta is left empty, can go-client serializer figure out GVK of this automatically?